src/page/Auth/FindPassword/ChangePassword/index.tsx

```diff
@@ -0,0 +1,77 @@
+import { useEffect } from 'react';
+import { useFormContext } from 'react-hook-form';
+import { useOutletContext } from 'react-router-dom';
+import { ChangePasswordForm } from 'model/auth';
+import { OutletProps } from 'page/Auth/FindPassword/entity';
+import { ReactComponent as Warning } from 'assets/svg/auth/warning.svg';
+import styles from 'page/Auth/FindPassword/index.module.scss';
+
+const passwordRegex = /^(?=.*[a-zA-Z])(?=.*\d)(?=.*[@$!%*?&])[A-Za-z\d@$!%*?&]{6,18}$/;
+
+export default function ChangePassword() {
+  const method = useFormContext<ChangePasswordForm>();
+  const {
+    register, formState: { errors, isValid }, getValues, clearErrors,
+  } = method;
```

Comment:
```suggestion
const {
    register, formState: { errors, isValid }, getValues, clearErrors,
  } = useFormContext<ChangePasswordForm>();
```
요렇게 해도 좋을 것 같은데 따로 분리하신 이유가 있을까요?

Reply:
꺼내쓰는 방식에 익숙해져서 습관처럼 이렇게 작성한 것 같습니다..
수정하겠습니다